backend-opal/src/server/stockServer.test.js

```diff
@@ -0,0 +1,26 @@
+jest.mock('./stockServer');
+const stockServer = require('./stockServer');
```

Comment:
@Yuan-EPAM here you should test the real service, not the mocked one, you should mock the dbDriver for testing the stockService and mock the stockService for testing the endpoint